dags/openshift_nightlies/tasks/utils/prom_mustgather.py

```diff
@@ -0,0 +1,94 @@
+import sys
```

Comment:
can this file be renamed to `scale_ci_diagnosis` or something similar to match the getter method at https://github.com/cloud-bulldozer/airflow-kubernetes/pull/22/files#diff-5c327263fbf41890b1c0c7f9db9da88277c03e04638966f33f08f4a4ed14090cR78

Reply:
done